apps/emqx_bridge_bigquery/src/emqx_bridge_bigquery_impl.erl

```diff
@@ -175,10 +188,26 @@ on_remove_channel(_ConnResId, ConnState, _ChanResId) ->
 on_get_channel_status(
     _ConnResId,
     ChanResId,
-    _ConnState = #{?installed_channels := InstalledChannels}
+    ConnState = #{?installed_channels := InstalledChannels}
 ) when is_map_key(ChanResId, InstalledChannels) ->
-    %% Should we attempt to get the table here?
-    ?status_connected;
+    #{?client := Client, ?project_id := ProjectId} = ConnState,
+    #{
+        ?table := Table,
+        ?dataset := Dataset,
+        ?request_ttl := RequestTTL
+    } = maps:get(ChanResId, InstalledChannels),
+    Opts = #{?request_ttl => RequestTTL},
+    case get_table(Client, ProjectId, Dataset, Table, Opts) of
+        {ok, _} ->
+            ?status_connected;
+        {error, #{status_code := 404}} ->
```

Comment:
nit, I guess the restapi response contains error messages, No? 

A bit risky that we may return misleading info to user.